purchase_work_acceptance_evaluation/views/work_acceptance_evaluation_views.xml

```diff
@@ -0,0 +1,71 @@
+<odoo>
+
+    <record id="view_work_acceptance_evaluation_list" model="ir.ui.view">
```

Comment:
Please declare id of all view follow this pattern https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#naming-xml-id